src/commands/stats.ts

```diff
@@ -30,6 +41,10 @@ export const stats: CommandDef = {
             name: 'Bot Online Time',
             value: 'Wake Time from JSON file'
           },
+          {
+            name: 'Version',
+            value: `[${commitHash}](https://github.com/bradtaniguchi/discord-bot-test/commit/${commitHash})`
```

Comment:
I was just thinking about having the sha shown as is, but having a link is better. Good idea!